awscli/handlers.py

```diff
@@ -108,6 +109,7 @@ def awscli_initialize(event_handlers):
     register_rds_modify_split(event_handlers)
     register_put_metric_data(event_handlers)
     register_ses_send_email(event_handlers)
+    cloudfront_register(event_handlers)
```

Comment:
These typically just get added at the end of the list.  Was there a specific reason it needs to be added here in the middle?